crates/remote/src/ssh_session.rs

```diff
@@ -59,13 +60,22 @@ pub struct SshSocket {
     socket_path: PathBuf,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Deserialize, Serialize)]
+pub struct SshPortForwardOption {
+    pub local_host: Option<String>,
+    pub local_port: u16,
+    pub remote_host: String,
+    pub remote_port: u16,
+}
+
```

Comment:
Feels a bit redundant to have the same structure in two crates with mapping between them, but they are slightly different with the `derive`. Should there just be one? And where is it better kept?